lib/state/ui/actions.ts

```diff
@@ -15,6 +15,13 @@ export const toggleSimperiumConnectionStatus: A.ActionCreator<A.ToggleSimperiumC
   simperiumConnected,
 });
 
+export const setSelectedNote: A.ActionCreator<A.SetSelectedNote> = (
+  note: T.NoteEntity | null
+) => ({
+  type: 'SET_SELECTED_NOTE',
+  note,
+});
```

Comment:
after these changes is there any reason to `setSelectedNote(null)`?

Reply:
also resolved in https://github.com/Automattic/simplenote-electron/pull/1851/commits/d14a6e9141ecbfc237edf5ec2c2abd50a5c2ad05